metadata/metadata.ts

```diff
@@ -513,7 +513,7 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
   "Christmas Tree": {
     name: "Christmas Tree",
     description:
-      "A rare collectible that provides holders with an airdrop on Christmas day.\n\n~~You can craft this item at the Goblin Blacksmith~~ **Sold out!**",
+      "A rare festive collectible. Place on your farm during the Festive Season and Santa may come to visit you! \n\n~~You can craft this item at the Goblin Blacksmith~~ **Sold out!** \n\n Tree must be placed on your farm.",
```

Comment:
Is it intentional to change the language here to `may`? Might appear to be changing the utility of the item. Could be
`Place on your farm during the Festive Season to get a guaranteed spot and Santa's nice list!`